lib/src/utils/plan.dart

```diff
@@ -98,16 +96,14 @@ class CBPlan{
     price = json['price'] as int;
     pricingModel = json['pricingModel'] as String;
     resourceVersion = json['resourceVersion'] as int;
-    setup_cost = json['setup_cost'] as int;
```

Comment:
This will be a breaking change right @cb-amutha?

Reply:
Reverted back the change and updated the PR.